csrc/scheduler/reduction_utils.cpp

```diff
@@ -357,7 +357,7 @@ void multiReductionInliner(
     TensorView* reference_tv,
     const bool unroll,
     const bool vectorize,
-    const bool is_outer_grid_persistence,
+    const bool use_grouped_reduction,
```

Comment:
nitpick: these names should also be updated IIUC.

Reply:
resolved offline. No need to update since it could be either iter_grouped or expr_grouped.